starter-code/scripts/articleView.js

```diff
@@ -16,50 +16,82 @@ articleView.populateFilters = function() {
   });
 };
 
+
 articleView.handleAuthorFilter = function() {
   $('#author-filter').on('change', function() {
-  if ($(this).val()) {
-      /* TODO: If the slect box changes to an option that has a value, we should:
-          1. Hide all of the articles
-          2. Fade in only the articles that match based on on the author
-            that was aselected. Hint: use an attribute selector to find
-            those articles that match the value, and then fade them in.
-        */
-  } else {
-    /* Otherwise, we should:
-        1. Show all the articles except the template */
-  }
+    if ($(this).val()) {
+      $('article').hide();
+      $(`article[data-author="${ $(this).val() }"]`).fadeIn(1000)
+      // $('article[data-author=" ' + $(this).val() + '"]').show();
+      //     /* TODO: DONE: If the slect box changes to an option that has a value, we should:
+      //         1. Hide all of the articles
+      //         2. Fade in only the articles that match based on on the author
+      //           that was aselected. Hint: use an attribute selector to find
+      //           those articles that match the value, and then fade them in.
+      //       */
+    } else {
+      $('article').not('.template').show();
+    }
     $('#category-filter').val('');
   });
 };
 
 articleView.handleCategoryFilter = function() {
-  /* TODO: Just like we do for #author-filter above, we should also handle
-  change events on the #category-filter element. Be sure to reset the
-  #author-filter while you're at it! */
+  $('#category-filter').on('change', function() {
+    if ($(this).val()) {
+      $('article').hide();
+      $(`article[data-category="${ $(this).val() }"]`).fadeIn(1000)
+
+      /* TODO: Done: Just like we do for #author-filter above, we should also handle
+      change events on the #category-filter element. Be sure to reset the
+      #author-filter while you're at it! */
+    } else {
+      $('article').show();
```

Comment:
You'll want to do something similar to above where you don't show (or explicitly hide) the .template and you should also be more specific in your selectors. See my comment on the handleArticleFilter method.